From 65672ed072d895361382478d82b33f8c11c7201d Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Sat, 10 Feb 2024 08:31:55 +0100
Subject: [PATCH 50/N] pathconv: don't skip arguments with double quote

It is used to pass strings/paths to the preprocessor and breaks
for example the CPython build. For example -DPREFIX='"/ucrt64"'.

Fixes #190
---
 winsup/cygwin/msys2_path_conv.cc | 1 -
 1 file changed, 1 deletion(-)

diff --git a/winsup/cygwin/msys2_path_conv.cc b/winsup/cygwin/msys2_path_conv.cc
index f6e91a2..4c0cc82 100644
--- a/winsup/cygwin/msys2_path_conv.cc
+++ b/winsup/cygwin/msys2_path_conv.cc
@@ -368,7 +368,6 @@ skip_p2w:
         switch (*it) {
         case '`':
         case '\'':
-        case '"':
         case '*':
         case '?':
         case '[':
